lua/entities/terminator_nextbot_zambiemecha/shared.lua

```diff
@@ -0,0 +1,248 @@
+AddCSLuaFile()
+
+ENT.Base = "terminator_nextbot_zambie"
+DEFINE_BASECLASS( ENT.Base )
+ENT.PrintName = "Mecha Zombie"
+ENT.Spawnable = false
+ENT.Author = "regunkyle"
+list.Set( "NPC", "terminator_nextbot_zambiemecha", {
+    Name = "Mecha Zombie",
+    Class = "terminator_nextbot_zambiemecha",
+    Category = "Nextbot Zambies",
+} )
+
+ENT.SpawnHealth = 1500
+ENT.WalkSpeed = 70
+ENT.MoveSpeed = 180
+ENT.RunSpeed = 300
+ENT.AccelerationSpeed = 1000
+ENT.FistDamageMul = 1.5
+ENT.MyPhysicsMass = 200
+ENT.JumpHeight = 200
+
+ENT.DoMetallicDamage = true
+ENT.MetallicMoveSounds = true
+ENT.ReallyStrong = true
+ENT.ReallyHeavy = true
+
+ENT.HasBrains = true
+ENT.IsStupid = false
+ENT.IsFodder = false
+
+ENT.term_SoundPitchShift = -30
+ENT.term_SoundLevelShift = 10
+
+ENT.Mecha_LastShockwave = 0
+ENT.Mecha_ShockwaveCooldown = 5
+
+ENT.TERM_MODELSCALE = 1.5
+
+ENT.term_LoseEnemySound = "npc/strider/strider_alert2.wav"
+ENT.term_CallingSound = "npc/strider/strider_alert5.wav"
+ENT.term_CallingSmallSound = "npc/strider/strider_alert6.wav"
+ENT.term_FindEnemySound = "npc/attack_helicopter/aheli_charge_up.wav"
+ENT.term_AttackSound = "npc/strider/strider_step5.wav"
+ENT.term_AngerSound = "npc/strider/strider_pain5.wav"
+ENT.term_DamagedSound = "npc/strider/strider_pain1.wav"
+ENT.term_DieSound = "npc/strider/strider_die1.wav"
+ENT.term_JumpSound = "npc/strider/strider_step6.wav"
+
+ENT.IdleLoopingSounds = {
+    "npc/strider/strider_ambient01.wav",
+}
+ENT.AngryLoopingSounds = {
+    "npc/attack_helicopter/aheli_rotor_loop1.wav",
+}

```

Comment:
Still not feeling the sounds, sounds too non-zombie and abstract, doesn't have that core
And the strider sounds are way too strong for a non-elite enemy